### PR TITLE
Plane front menu reorg

### PR DESCRIPTION
**Pull request by Hwurzburg**
updated to match recent Copter reorg
ancient intro page updated

### CODE REVIEW

File: plane/source/docs/additional-information.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+.. toctree::
+    :maxdepth: 1
+
+    Upcoming Features <common-master-features>
+    Antenna Tracking <common-antenna-tracking>
+    Simulation <common-simulation>
+    OEM Customization <common-oem-customizations>
+    Use Cases and Applications<common-use-cases-and-applications>
```
Comment (brianmusslewhite) [inline plane/source/docs/additional-information.rst:15]: I think this needs a space

File: plane/source/index.rst [hunk 2]
```diff
@@ -105,23 +105,16 @@ Monitor https://discuss.ardupilot.org/c/arduplane for plane-related announcement
...
    :hidden:
    
-    Introduction <docs/introduction>
+    Introduction <docs/plane-introduction>
     AutoPilot Hardware <docs/common-autopilots>
+    Ground Control Stations <docs/common-GCS>
     First Time Setup <docs/arduplane-setup>
     First Flight and Tuning <docs/first-flight-landing-page>
     QuadPlane Setup and Operation <docs/quadplane-support>
+    Mission Planning <docs/common-mission-planning>
```
Comment (brianmusslewhite) [inline plane/source/index.rst:114]: I noticed this is also under Ground Control Stations. Do you think it belongs better there or on the 1st level?

Comment (Hwurzburg) [reply plane/source/index.rst:114]: both places....Randy wanted it on the left....and we may add to the GCS landing page

File: plane/source/docs/additional-information.rst (hunk 1 above)
Comment (Hwurzburg) [reply plane/source/docs/additional-information.rst:15]: not needed.....it is needed on https:/ links...ie  `this link <URL>`__

**Pull request merged**